scripts/api.sh

```diff
@@ -206,15 +218,27 @@ stageApi()
     buildApp $CONTAINER
   else
     docker stop -t 0 $NAME >/dev/null
+    docker rm $NAME >/dev/null
+    docker pull $CONTAINER
   fi
 
   if [ ! "$UPDATE" = true ]; then
     startDB $DB_NAME
-
     if [ -z "$(docker ps | grep $NAME)" ]; then
       printf "\n"
       echo "Starting API..."
-      docker run -d --rm -p $PORT:443 -v $BUILD_DIR:$CLIENT_VOLUME -v $BUILD_DIR:$TEMPLATE_CLIENT_VOLUME -v $TEMPLATE_DIR:$TEMPLATE_REGISTRATION_VOLUME -e QPC_DBMS_HOST=$DB_NAME --link $DB_NAME:qpc-link --name $NAME $CONTAINER >/dev/null
+      docker run -d -p $PORT:443 \
```

Comment:
request:
... you drop the `--rm` back in the `docker run -d --rm -p $PORT:443` it appears to actually stop the Docker container, at least on Mac OS when you ctrl-c out of the terminal process